pgnReader.py

```diff
@@ -0,0 +1,38 @@
+'''
+    path - path to .pgn file
+    maxGame - maximum number of games to parse
+'''
+def pgnParse(path,maxGame=None):
+    result=[{}]
+    with open(path,'r') as f:
+        for line in f:
+            #tags begin with "["
+            #Movetext begin with "1."
+            if(line[0]=="["):
+                j=line.split(" ")

```

Comment:
j is a good variable name for a loop index, but for nothing else - pls rename this to something more descriptive